packages/fxa-auth-server/scripts/recorded-future/lib.ts

```diff
@@ -23,13 +29,15 @@ export const createCredentialsSearchFn =
     if (error) {
       // the Recorded Future Identity OpenAPI definition does not include any
       // response information aside from the http 200 response
-      throw new Error(`Recorded Future error response: ${error}`);
+      throw new Error(
+        `Recorded Future credentials search error: ${JSON.stringify(error)}`
```

Comment:
Provide error cause instead of stringifying?


```suggestion
        `Recorded Future credentials search error: ${error.message}`, { cause: error }
```

Reply:
The `error` response is not an Error object.  Let me try deliberately cause an error to see what we can do and update again in a future PR.  The reason I made this change is that I saw `[object Object]` when I encountered a transient error with the Recorded Future API yesterday.